Swift Vocabulary/Swift Vocabulary/WordsTableViewController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  WordsTableViewController.swift
+//  Swift Vocabulary
+//
+//  Created by Ian Becker on 5/11/20.
+//  Copyright © 2020 Ian Becker. All rights reserved.
+//
+
+import UIKit
+
+class WordsTableViewController: UITableViewController {
+    
+    // Data source for the table view
+    var vocabWords: [VocabularyWord] = [VocabularyWord(word: "Boolean", definition: "A data type that stores either true or false."), VocabularyWord(word: "Class", definition: "A custom data type that can have one or more properties and one or more methods. Reference type."), VocabularyWord(word: "Double", definition: "A high-performance, unordered collection of values stored using a key for fast access."), VocabularyWord(word: "Method", definition: "A function that belongs to a struct or class."), VocabularyWord(word: "String", definition: "A collection of letters.")]
```

Comment:
This is an incredibly long line. It's good practice that after the comma that separates the values in the array, press Enter that way each value is on its own line. It makes things easier to read.

Reply:
Got it. Thank you. I tried to use the auto indent/format thing but I guess it didn't work!